### ISSUE TITLE
JAWS announces radio button 1 of 1 in Chrome for implicitly wrapped radio buttons that also contain spans 

### ISSUE THREAD

**Issue by mraccess77** [FreedomScientific/VFO-standards-support#575]
When radio buttons in a group have the same name (as they should) and are wrapped with implicit labels (acceptable method) and also contain spans within the label - the radio buttons are announced in Chrome as 1 of 1 for each.  This issue seems to occur for NVDA as well.  It does not occur in Firefox.

### Expected result

JAWS should announce radio button 1 of 2 and 2 of 2.

### Actual result

JAWS indicates radio button of 1 of 1

**Comment by stevefaulkner** [comment]
@mraccess77 can you provide a test case?

**Comment by mraccess77** [comment]
Sorry, I had pasted in the code but Github keeps rendering it.  I have put up a pen here: https://codepen.io/mraccess/pen/MWoLOMX

**Comment by stevefaulkner** [comment]
@mraccess77 I had a look at the accessibility tree and it is a chrome bug, the `setsize` and `posinset` values for each of the `radio` buttons is `1`. In Firefox they are reported correctly. So JAWS is correctly announcing the incorrect information exposed by Chrome.

**Comment by JAWS-test** [comment]
The problem always occurs as soon as one of the radio buttons has another child element in the label, regardless of the role of the child element (e.g. a graphic in the label is sufficient).

**Comment by stevefaulkner** [comment]
chrome bug filed https://bugs.chromium.org/p/chromium/issues/detail?id=1254723
